.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -1,15 +1,9 @@
 package configurations
 
-import jetbrains.buildServer.configs.kotlin.v2017_2.BuildFeatures
-import jetbrains.buildServer.configs.kotlin.v2017_2.BuildStep
-import jetbrains.buildServer.configs.kotlin.v2017_2.BuildSteps
-import jetbrains.buildServer.configs.kotlin.v2017_2.BuildType
-import jetbrains.buildServer.configs.kotlin.v2017_2.CheckoutMode
-import jetbrains.buildServer.configs.kotlin.v2017_2.FailureAction
-import jetbrains.buildServer.configs.kotlin.v2017_2.ProjectFeatures
-import jetbrains.buildServer.configs.kotlin.v2017_2.buildFeatures.commitStatusPublisher
-import jetbrains.buildServer.configs.kotlin.v2017_2.buildSteps.GradleBuildStep
-import jetbrains.buildServer.configs.kotlin.v2017_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2018_1.*
```

Comment:
Let's not do a `*` import.

Reply:
Done.

Reply:
Hmm... This is still a `*` import... Do you maybe need to update your IDEA settings?

Reply:
It might help if I actually push my changes...